backend/app/utils/configs/validations.py

```
@@ -56,7 +84,7 @@ def validate_config_data(config_definition_key: str, data: dict) -> None:
     try:
         jsonschema.validate(data, json_schema)
     except jsonschema.ValidationError as e:
-        raise ValueError(f"Configuration schema mismatch: {e.message}")
+        raise validation_error("data", e.message.split("\n", 1)[0])
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)